README.md

```diff
@@ -46,7 +66,176 @@ Pipelines:
 export JF_PRODUCT_DATA_INTERNAL=/opt/jfrog/pipelines/var/
```

Comment:
Isn't this /var/opt/jfrog/<product_name> ?

Reply:
No, this is the location where logs for pipelines are stored